### PR TITLE
Bc1 7 base structure for app land

### PR DESCRIPTION
**Pull request by MaxRichard2007**

### CODE REVIEW

File: AppLand/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (MichaelAndish) [inline AppLand/index.html:1]: https://weprodev.atlassian.net/wiki/spaces/B1/pages/491547/Front-end+Code+Convention#2.-HTML-Standards

File: AppLand/scss/_button.scss
Comment (MichaelAndish) [inline AppLand/scss/_button.scss:1]: As we discussed we'll create these modules in shared directory

File: AppLand/scss/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    $BLACK          : rgb(000, 000, 000); // #000
+    $WHITE          : rgb(255, 255, 255); // #fff
+
+    $ShadowBlack    : rgb(024, 026, 027); // #181a1b
+    $VibrantGreen   : rgb(084, 234, 084); // #54ea54
+    
+    $LimeGreen      : rgb(071, 198, 071); // #47c647
+    $LightGray      : rgb(170, 170, 170); // #aaa
+
+    $TransparentBlack : rgba(0, 0, 0, 0.5); // shadow Black
```
Comment (MichaelAndish) [inline AppLand/scss/_variables.scss:26]: please, follow the code convention rules

File: AppLand/scss/main.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    */
+
+    @import "button.scss";
+
+    /*--------------------------------
+    ***  shared files 
+    **  
+    */   
+
+    @import "../../shared/style.css";
```
Comment (MichaelAndish) [inline AppLand/scss/main.scss:42]: style.css or style.scss ?!

File: shared/scss/abstracts/_media-query.scss [hunk 4]
```diff
@@ -15,8 +15,6 @@
 	// @import "compass/css3";
 	// @import "compass/utilities/general/clearfix";
```
Comment (MichaelAndish) [inline shared/scss/abstracts/_media-query.scss:17]: remove comments if you don't want to use it

File: shared/scss/import.scss
Comment (MichaelAndish) [inline shared/scss/import.scss:1]: I recommend to rename this file to sth like **shared.scss** 
in each project you can define which files do you want to import

**Review by MichaelAndish — CHANGES_REQUESTED** [review]

File: AppLand/scss/main.scss (hunk 3 above)
Comment (MaxRichard2007) [reply AppLand/scss/main.scss:42]: I changed it to shared.css
Yes, this is a css file

File: shared/scss/import.scss
Comment (MaxRichard2007) [reply shared/scss/import.scss:1]: OK

File: shared/scss/abstracts/_media-query.scss (hunk 4 above)
Comment (MaxRichard2007) [reply shared/scss/abstracts/_media-query.scss:17]: OK

File: AppLand/scss/_variables.scss (hunk 2 above)
Comment (MaxRichard2007) [reply AppLand/scss/_variables.scss:26]: How should I write?

File: AppLand/scss/_button.scss
Comment (MaxRichard2007) [reply AppLand/scss/_button.scss:1]: OK, I will remove it

File: AppLand/scss/_variables.scss (hunk 2 above)
Comment (MichaelAndish) [reply AppLand/scss/_variables.scss:26]: read code convention, if you don't know ask the team

File: AppLand/scss/main.scss (hunk 3 above)
Comment (MichaelAndish) [reply AppLand/scss/main.scss]: I know it's css I can read it, 
you import CSS in SCSS ?! do you think it's correct

File: AppLand/scss/main.scss [hunk 5]
```diff
@@ -26,20 +26,13 @@
...
-    @import "button.scss";
-
+    
     /*--------------------------------
     ***  shared files 
     **  
     */   
 
-    @import "../../shared/style.css";
+    @import "../../shared/shared.css";
```
Comment (MichaelAndish) [inline AppLand/scss/main.scss:35]: in the SCSS you should import SCSS NOT CSS

File: shared/scss/abstracts/_media-query.scss [hunk 6]
```diff
@@ -7,14 +7,9 @@
 
 	/*--------------------------------
 	***  GENERAL DIFINITIONS
-	**	 COMPASS , SUSY , CSS3 , CLEARFIX
+	**	 SUSY , CSS3 , CLEARFIX
```
Comment (MichaelAndish) [inline shared/scss/abstracts/_media-query.scss:10]: I didn't know you are using SUSY

**Review by MichaelAndish — CHANGES_REQUESTED** [review]

**Review by MichaelAndish — APPROVED** [review]

**Review by abdolmajed3 — APPROVED** [review]
well done

**Pull request merged**